lib/rules/max_line_length.spec.ts

```diff
@@ -16,7 +16,7 @@ describe('max_line_length rule', () => {
 			error = rule.check({ max_line_length: 2 }, fooLine);
 			expect(error).to.be.ok;
 			expect(error.rule).to.equal('max_line_length');
-			expect(error.message).to.be.equal('line length: 3, exceeds: 2');
+			expect(error.message).to.be.equal('invalid line length: 3, expected: 2');
```

Comment:
This is not actually true. We don't "expect" a line length of `80` just because they have the `max_line_length = 80` rule. That's why I chose the word "exceeds" here, because any invalid `max_line_length` will be a case where one "exceeds" the defined `max_line_length`.